src/main/webapp/app/exercises/shared/dashboards/tutor/exercise-assessment-dashboard.component.ts

```diff
@@ -601,7 +605,14 @@ export class ExerciseAssessmentDashboardComponent implements OnInit {
     viewComplaint(complaint: Complaint) {
         const submission: Submission = complaint.result?.submission!;
         // numberOfAssessmentsOfCorrectionRounds size is the number of correction rounds
-        this.openAssessmentEditor(submission, this.numberOfAssessmentsOfCorrectionRounds.length - 1);
+        if (complaint.complaintType === ComplaintType.MORE_FEEDBACK) {
+            this.openAssessmentEditor(submission, this.numberOfAssessmentsOfCorrectionRounds.length - 1);
+        }
+        const submissionToView = this.submissionsWithComplaints.filter((dto) => dto.submission.id === submission.id).pop()?.submission;
+        if (submissionToView) {
+            submissionToView.results = submissionToView.results?.filter((result) => result.assessmentType !== AssessmentType.AUTOMATIC);
+            this.openAssessmentEditor(submissionToView, submissionToView.results!.length - 1);
```

Comment:
`submissionToView.results` might be null or undefined at this point! (Otherwise why the `results?` above.)

Reply:
Thats actually a good point

Reply:
I added an additional check